provd/phoned_user_service.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+URL_FORMAT = u'{scheme}://{hostname}:{port}/0.1/{vendor}/user_service/{service}?user_uuid={user_uuid}&enabled={enabled}'
```

Comment:
I feel like we should allow the prefix to be configured here.

Reply:
What do you mean by the prefix? The part before the API version?